process/slash/notifier/slashingNotifier.go

```diff
@@ -179,12 +179,8 @@ func (sn *slashingNotifier) selectKeyPair() KeyPair {
 	var shardID uint32
 	selfID := sn.shardCoordinator.SelfId()
 
-	first := true
 	for currShardID := range sn.keyPairs {
-		if first == true {
-			shardID = currShardID
-			first = false
-		}
+		shardID = currShardID
 
 		if currShardID != selfID {
 			shardID = currShardID
```

Comment:
this is no longer needed if initialized on top with currShardID

Reply:
Good catch, removed the extra line